cloud_images/centos7/packer.json

```diff
@@ -22,18 +22,50 @@
       {
         "device_name": "/dev/sde",
         "volume_type": "gp2",
-        "volume_size": 20,
+        "volume_size": 50,
         "delete_on_termination": true
       },
       {
         "device_name": "/dev/sdf",
         "volume_type": "gp2",
+        "volume_size": 100,
+        "delete_on_termination": true
+      },
+      {
+        "device_name": "/dev/sdg",
+        "volume_type": "gp2",
+        "volume_size": 50,
+        "delete_on_termination": true
+      },
+      {
+        "device_name": "/dev/sdh",
+        "volume_type": "gp2",
         "volume_size": 20,
         "delete_on_termination": true
+      }
+    ],
+    "launch_block_device_mappings": [
```

Comment:
That was something @lingmann added when we were debugging the missing EBS devices on some of the nodes that got spun up. I believe he got it from our Teamcity AMI. It had something to do with when the device would be available to the system. @lingmann?